pkg/download/bucket/bucket.go

```diff
@@ -69,17 +77,16 @@ func (d *Downloader) convertAllObjects(projectName string, objects [][]byte) []c
 
 		c, err := convertObject(o, projectName)
 		if err != nil {
-			log.WithFields(field.Type("bucket"), field.Error(err)).
-				Error("Failed to decode API response objects for bucket resource: %v", err)
-			continue
-		}
+			if errors.As(err, &skipErr{}) {
+				log.WithFields(field.Type("bucket")).
+					Debug("Skipping bucket: %s", err.Error())
+			} else {
+				log.WithFields(field.Type("bucket"), field.Error(err)).
+					Error("Failed to decode API response objects for bucket resource: %v", err)
+			}
 
-		// exclude builtin bucket names
-		if tools.IsDefault(c.OriginObjectId) {
```

Comment:
IsDefault is used on deletion as well. Can we use the updatable property there as well and replace our name-based logic?

Reply:
Could do. For now, I've reintroduced the IsDefault method, as older versions of Dynatrace may not set the 'new' field. Just to be sure.
For delete, we could also introduce the additional updatable check.